lib/statsig/configs.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Statsig.Configs do
+  use GenServer
+  require Logger
+
+  @table_name :statsig_configs
+
+  defmodule State do
+    @type t :: %__MODULE__{
+            last_sync_time: integer(),
+            reload_timer: reference() | nil,
+            reload_interval: integer()
+          }
+
+    defstruct last_sync_time: 0,
+              reload_timer: nil,
+              reload_interval: 10_000
```

Comment:
nit: `:timer.seconds(10)` instead of `10_000`. 
You can also make a module attribute and use that
```elixir
@default_reload_interval :timer.seconds(10)
```
then replace everywhere you have `10_000` with `@default_reload_interval`